src/commands/game/queue/queue.ts

```diff
@@ -7,12 +7,13 @@ const commands = ['queue', 'startqueue'];
 
 export class QueueAction extends BaseCommandClass {
   static instance: QueueAction = new QueueAction(commands);
-  queue = [];
+  queue: User[] = [];
```

Comment:
With this change, my previous proposal for how adding someone else to the queue might not work very well.